package.json

```diff
@@ -29,30 +29,30 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.2.0",
-    "ember-cli": "^2.7.0",
+    "broccoli-asset-rev": "^2.4.2",
+    "ember-ajax": "^2.0.1",
```

Comment:
No problem.  I noticed that `ember-data` is a dependency that we do not use either.  Should I create an issue for it's removal?

I have reverted this change.